Source/Libraries/Adapters/GrafanaAdapters/GrafanaDataSourceBase_AncillaryOperations.cs

```diff
@@ -316,16 +327,26 @@ private static Task ProcessDistribution<T>(List<DataSourceValueGroup<T>> queryVa
 
         return Task.Factory.StartNew(() =>
         {
+
+
             List<MetadataMap[]> groupedMaps = [];
             List<MetadataMap> matchingMaps = [metadataMaps[0]];
+            List<MetadataMap> mapsWithGroupBy = [];
             MetadataMap firstGroupMap = metadataMaps[0];
-
             // Organize metadata maps with overlapped coordinates into groups, this code
             // assumes that metadata maps are already sorted by longitude and latitude:
             for (int i = 1; i < metadataMaps.Length; i++)
```

Comment:
Add a line feed to separate the variable declarations from comment associated with the next for statement.